docs/contributing-analyzer.md

```diff
@@ -0,0 +1,59 @@
+# Building, Testing and Debugging the Analyzer
+
+All C# and VB.NET analyzers present in SonarLint for Visual Studio, in the SonarQube SonarC# Plugin or in the
+SonarQube SonarVB plugin are being developed here.
+These analyzers rely on Roslyn 1.0 API.
+
+## Working with the code
+1. Clone SonarC# repository: https://github.com/SonarSource/sonar-csharp.git
+2. Go to `sonaranalyzer-dotnet` sub-folder
+3. Open SonarAnalyzer.sln
+
+Note: SonarAnalyzer.VS2015.sln and SonarAnalyzer.VS2017.sln are not ready for use yet and are part of our plan to support
+.Net Core and C# 7 features.
+
+## Developing with Visual Studio
+We currently rely on Visual Studio 2015 for the development on the Analyzer. Any change to the solution or the projects
+linked to development with Visual Studio 2017 should not be included in PR.
+
+## Running Tests
+
+### Unit Tests
+You can run the Unit Tests via Test Explorer of Visual Studio.
+
+### Integration Tests
+If this is the first time you are running the ITs you will need to run the following command:
+git submodule update --init --recursive
+
+To run the ITs you will need to follow this pattern:
+1. Open the `Developer Command Prompt for VS2015`
```

Comment:
Similar to blank lines before section headings, I also recommend to put a blank line between paragraphs and lists